docs/source/usage/plugins/countPerSupercell.rst

```diff
@@ -14,7 +14,7 @@ The plugin is available as soon as the :ref:`libSplash and HDF5 libraries <insta
 .cfg files
 ^^^^^^^^^^
 
-By specifying the perodicity of the output using the comand line argument ``--e_macroParticlesPerSuperCell.period`` (here for an electron species ``e``) with picongpu, the plugin is enabled.
+By specifying the periodicity of the output using the command line argument ``--e_macroParticlesPerSuperCell.period`` (here for an electron species ``e``) with picongpu, the plugin is enabled.
```

Comment:
Not that important, but it is probably more precise to call ``--e_macroParticlesPerSuperCell.period`` an option, not an argument, as the argument is the value that comes after it.

Reply:
We keep command line argument here, since we have `.cfg` files (command line options) and `.param` files (compile time options)

Reply:
I see, then ofc it makes sense to keep as is (although this is illogical naming to me).

Reply:
One has to see this in consistency with the introduced terms of the intro chapters: https://picongpu.readthedocs.io/en/0.4.0-rc2/usage/basics.html

Reply:
Ok, I think [this](https://stackoverflow.com/a/36495940) is a good explanation for distinction between options, parameters, and arguments. It is almost the same as in the basic section.

Reply:
In order to keep consistency, I will keep it as is.